code/game/mecha/equipment/tools/tools.dm

```diff
@@ -1186,3 +1187,99 @@
 
 #undef LOCKED
 #undef OCCUPIED
+
+/obj/item/mecha_parts/mecha_equipment/tool/jetpack
+	name = "ion jetpack"
+	desc = "Using directed ion bursts and cunning solar wind reflection technique, this device enables controlled space flight."
+	icon_state = "mecha_jetpack"
+	equip_cooldown = 5
+	energy_drain = 50
+	var/wait = 0
+	var/datum/effect/effect/system/ion_trail_follow/ion_trail
+
+
+	/obj/item/mecha_parts/mecha_equipment/tool/jetpack/can_attach(obj/mecha/M as obj)

```

Comment:
Everything here and down.. Is indented when it should not be.

Reply:
dammnit